docs/api-reference/geo-layers/mvt-layer.md

```diff
@@ -117,38 +117,40 @@ On top of the [default options](/docs/api-reference/core/layer.md#loadoptions),
 
 - [MVTLoader](https://loaders.gl/modules/mvt/docs/api-reference/mvt-loader)
 
+## Methods
 
-### Callbacks
+##### `getVisibleTiles`
+
+`getVisibleTiles` retrieves the visible tiles in the current viewport.
+
+## Callbacks
```

Comment:
The callbacks are also layer properties. Move the methods out of the properties section.